secrets/interfaces.go

```diff
@@ -49,9 +48,9 @@ type SecretClient interface {
 	// SetAuthToken sets the internal Auth Token with the new value specified.
 	SetAuthToken(ctx context.Context, token string) error
 
-	// GetKeys retrieves the keys at the provided sub-path. Secret Store returns an array of keys for a given path when
+	// GetSecretNames retrieves the keys at the provided secretName. Secret Store returns an array of keys for a given secretName when
 	// retrieving a list of keys, versus a k/v map when retrieving secrets.
-	GetKeys(subPath string) ([]string, error)
+	GetSecretNames(secretName string) ([]string, error)
```

Comment:
No parameter needed as it gets all the secret names from the base path for the Store
```suggestion
	GetSecretNames() ([]string, error)
```